rom/dev/tools/test-rt/src/main.rs

```diff
@@ -38,6 +40,10 @@ const BANNER: &str = r#"
 pub extern "C" fn rt_entry() -> ! {
     cprintln!("{}", BANNER);
 
+    for (x, item) in PAD.iter().enumerate() {
+        cprint!("PAD[{}] = 0x{:08X}", x, *item);
```

Comment:
is this print only here to prevent the compiler from optimizing the padding array out?

Reply:
Yes.